harvard-lts/CURIOSity
# added comma delimiter to search resulsts

**Ltsmaint752 add commas to search results**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/LTSMAINT-760

# What does this Pull Request do?
Adds a comma delimiter to search results. Screenshots of behavior attached. Since harvested test data does not have enough results to test functionality, I filled in large numbers in the front-end file to test that the delimiter worked properly.

# How should this be tested?
Do a search with over and under 1000 results and ensure that display is correct.

# Screenshots:
Large number before adding delimiter to code (current functionality):
<img width="1134" alt="Screen Shot 2023-06-16 at 12 45 00 PM" src="https://github.com/harvard-lts/CURIOSity/assets/46659222/78123479-ac82-4c63-be3b-7d30aeeecffe">
Large number after adding delimiter:
<img width="1134" alt="Screen Shot 2023-06-16 at 12 53 57 PM" src="https://github.com/harvard-lts/CURIOSity/assets/46659222/623c7b6e-1123-4dca-b20b-d2336a13f00c">
Testing small number with delimiter to ensure it still works properly:
<img width="1134" alt="Screen Shot 2023-06-16 at 1 07 20 PM" src="https://github.com/harvard-lts/CURIOSity/assets/46659222/ceb75bc8-bfa0-4b87-9790-8f2235b507aa">
